crates/loans/src/lib.rs

```diff
@@ -343,6 +346,40 @@ pub mod pallet {
         IncentiveReservesReduced(T::AccountId, AssetIdOf<T>, BalanceOf<T>),
     }
 
+    #[pallet::hooks]
+    impl<T: Config> Hooks<T::BlockNumber> for Pallet<T> {
+        fn on_runtime_upgrade() -> frame_support::weights::Weight {
```

Comment:
Please add a guard that checks (and updates) the `palletVersion` so this is only executed once. Note that `palletVersion` is part of Frame - we want to move away from the `StorageVersion` enum that we were using until now

Reply:
Hmm yea there is advantage of using the storage version, namely that you can easily see if the migration runs or not. But fair enough about it already been guaranteed to only run once

Reply:
We can check whether the migration ran if the `MinExchangeRate` and `MaxExchangeRate` storage items are non-zero